dns_scripts/GoDaddy-README.txt

```diff
@@ -1,3 +1,11 @@
+---------------------------------------------------------------------------
+  NOTE: GoDaddy updated their domain API requirements in 2024! It's now
+  required to have at least 10-50 domains in your account. Another option
+  is to purchase the "Discount Domain Club" to gain access to the API.
```

Comment:
Just ran in to this with GoDaddy and purchased the Discount Domain Club membership, only to find out from them that it wasn't good enough and we needed the "Premium" discount domain club.  Never looked any further in to this and abandonded GoDaddy in favor of Route 53, but you may want to indicate that Premium is required.

Reply:
Thanks, updated.